packages/api/src/command/webhook/webhook.ts

```diff
@@ -23,7 +23,12 @@ type WebhookPingPayload = {
   ping: string;
 };
 
-export type WebhookMetadataPayload = { messageId: string; when: string; type: string };
+export type WebhookMetadataPayload = {
+  messageId: string;
+  when: string;
+  type: string;
+  data?: unknown;
```

Comment:
Suggestion: can we add a TSDoc for the `WebhookMetadataPayload` type, please? It is growing and now it seems a good time to indicate what each property means - especially `data`.

Reply:
TSDoc added